app/src/main/res/layout/activity_main2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this xml is added by mistake, please remove it if it's the case

Reply:
Thanks!